po/fr.po

```diff
@@ -640,13 +638,13 @@ msgstr "1. Décompresser %s"
 msgid "2. Copy the contents into %s."
 msgstr "2. Copier le dossier obtenu dans %s."
 
-#, fuzzy, tcl-format
+#, tcl-format
 msgid "[deken] uninstalling '%s'"
-msgstr "[deken]: connecté ? %s"
+msgstr "[deken] désinstallation de %s"
 
 #, tcl-format
 msgid "Uninstalling %1$s from %2$s failed!"
-msgstr ""
+msgstr "Impossible de désinstaller %1$s de %2$s !"
```

Comment:
Suggesting: "Échec de la désinstallation de %1$s de %2$s !"

Reply:
Yes, it's better to stick to the original.

Reply:
:+1: